### DISCUSSION TITLE
Autofill product from external source

### DISCUSSION THREAD

**Discussion by fusionconsulting** [lunarphp/lunar#314] [Q&A]
I would like to be able to populate certain fields of a product from an external source. So, the idea would be that the user enters a product code which is loaded from an external source and populates things like title, description, price etc automatically.

Firstly, is this possible without modifying the package source? If so, how would I go about it?

**Comment by glennjacobs** [comment]
I would probably make a custom screen for this, that looks up the product from the external source and creates a product in GetCandy, then redirects to the product editing screen.

**Reply by fusionconsulting** [reply]
Ok, thanks. That sounds like a reasonable solution. Is there a way to replace the create product button in the product list to go to the custom page? That would make it a perfect solution.

**Reply by glennjacobs** [reply]
Right now, no it's not possible. But we do have something in development that will allow this and more in the future.

**Reply by fusionconsulting** [reply]
Great, I'll be happy to test that when available. Can I add a button next to it perhaps? What would be the best way to add a link to this function? I know I can add to the side menu but I feel like this would be better placed somewhere on the index screen.

**Reply by glennjacobs** [reply]
That's the thing, there isn't a way at the moment to add to those screens. We need to add in places for developers to extend.
